vsts/pipelines/build-image-platforms/python.yml

```diff
@@ -0,0 +1,94 @@
+variables:
+  ascName: OryxMCR
+  acrName: oryxdevmcr.azurecr.io
+
+queue:
+  name: OryxLinux
+
+trigger:
+  branches:
+    include:
+    - master
+  paths:
+    include:
+    - images/build/python
+    - build/
+    - vsts/
+
+steps:
```

Comment:
Take a look at my other comment. If you have a script which builds all these images, then this script could be used locally as well as in this yml file. This is similar approach to how existing `build-runtimeimages.sh` would do

Reply:
Good point. However, when I think about it, the only thing we will save developers, is the need to run `docker build -f <path> .`.
Not a lot of work, and not very different from running `./build-base-platforms.sh`.
WDYT?

Reply:
I feel its just convenient to call into a script and let it handle the details. Also calling the `docker build` command has its own issues like from which directory you would be executing it because the docker build context would be from that directory and sometimes people can get it wrong. We also wouldn't be having different approaches while building locally or in vsts.